### PR TITLE
Add option to clear activity on app termination

### PR DESCRIPTION
**Pull request by JulianBissekkou**

### CODE REVIEW

File: ios/Classes/SwiftLiveActivitiesPlugin.swift [hunk 1]
```diff
@@ -1,91 +1,97 @@
...
+    if self.appGroupId == nil || self.sharedDefault == nil {
       result(FlutterError(code: "NEED_INIT", message: "you need to run 'init()' first with app group id to create live activity", details: nil))
     }
   }
 
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
     if #available(iOS 16.1, *) {
-      switch (call.method) {
+        
+      switch call.method {
```
Comment (JulianBissekkou) [inline ios/Classes/SwiftLiveActivitiesPlugin.swift:40]: What are you using to format your swift code? I had problems getting the formatting right when editing the files in xcode 🤔

**Comment by JulianBissekkou** [comment]
Again: Just draft, will do some more testing tomorrow, but I would love to get some feedback on the changes that I did on the dart side.

File: ios/Classes/SwiftLiveActivitiesPlugin.swift [hunk 2]
```diff
@@ -133,21 +139,26 @@ public class SwiftLiveActivitiesPlugin: NSObject, FlutterPlugin, FlutterStreamHa
...
-        if (activityId == activity.id) {
+        if activityId == activity.id {
           for item in data {
             sharedDefault!.set(item.value, forKey: item.key)
           }
+            
           
+          // Why is appGroupId empty here? Maybe add a comment.
           let updatedStatus = LiveActivitiesAppAttributes.LiveDeliveryData(appGroupId: "")
           await activity.update(using: updatedStatus)
```
Comment (JulianBissekkou) [inline ios/Classes/SwiftLiveActivitiesPlugin.swift:163]: Can you maybe explain why the appGroupid is empty here?

File: ios/Classes/SwiftLiveActivitiesPlugin.swift (hunk 1 above)
Comment (istornz) [reply ios/Classes/SwiftLiveActivitiesPlugin.swift:40]: I notice that "Indent switch/case labels" was not active on my Xcode for Swift...
I enabled it 👍

File: ios/Classes/SwiftLiveActivitiesPlugin.swift (hunk 2 above)
Comment (istornz) [reply ios/Classes/SwiftLiveActivitiesPlugin.swift:163]: Seems to be an oversight...

Comment (istornz) [reply ios/Classes/SwiftLiveActivitiesPlugin.swift:163]: We need to remplace the empty string with ```self.appGroupId``` 👍

**Review by istornz — APPROVED** [review]

File: ios/Classes/SwiftLiveActivitiesPlugin.swift (hunk 1 above)
Comment (JulianBissekkou) [reply ios/Classes/SwiftLiveActivitiesPlugin.swift:40]: No i usually just go with "Structure" -> "Reindent". I did that and pushed the changes. The diff is huge, but I didn't added anything else besides the change mentioned in the other discussion 👍🏽

File: ios/Classes/SwiftLiveActivitiesPlugin.swift (hunk 2 above)
Comment (JulianBissekkou) [reply ios/Classes/SwiftLiveActivitiesPlugin.swift]: Pushed the change 👍🏽

**Comment by JulianBissekkou** [comment]
@istornz I pushed the formatting and "AppGroupId on update" change. Please take a last look and feel free to merge the PR after that 👍🏽

**Pull request merged**